src/app/search-bar/search-bar.component.css

```diff
@@ -62,4 +78,4 @@
   .align-search-btn{
     left: -5%;
   }
-}
+}
```

Comment:
@harshit98  i was not able to get any idea where sub menu is defined in the code . rest changes have been implemented .